clj-refactor.el

```diff
@@ -2171,13 +2171,13 @@ FEATURE is either :clj or :cljs."
 
 (defun cljr--get-aliases-from-middleware ()
   (-when-let (aliases (cljr--call-middleware-for-namespace-aliases))
-    (if (cljr--clj-context?)
+    (if (cljr--clj-context-p)
         (gethash :clj aliases)
       (gethash :cljs aliases))))
 
 (defun cljr--is-global-alias (alias)
```

Comment:
`cljr--global-alias-p`, though I'd prefer we rename this altogether.  When reading the code it wasn't at all obvious what it was until I saw the implementation.  Perhaps `cljr--cljs-js-alias-p` or just `cljr--js-alias-p`?